From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sun, 26 May 2024 03:48:20 -0500
Subject: [PATCH] Fix build with llvm-18

No idea why this isn't biting other distros, but on Solus this fails with
```
ld.lld: error: version script assignment of 'global' to symbol 'test_shade' failed: symbol not defined
```
Fix this by removing that from the version script.

---
 src/build-scripts/hidesymbols.map | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/build-scripts/hidesymbols.map b/src/build-scripts/hidesymbols.map
index 00ad2128..7f29c2c8 100644
--- a/src/build-scripts/hidesymbols.map
+++ b/src/build-scripts/hidesymbols.map
@@ -1,4 +1,4 @@
 {
-    global: *OSL*; test_shade; *osl*imageio*; *osl_input_extensions;
+    global: *OSL*; *osl*imageio*; *osl_input_extensions;
     local: osl_*; *pvt*; *;
 };
